rasa_nlu/evaluate.py

```diff
@@ -477,7 +494,7 @@ def run_cv_evaluation(data, n_folds, nlu_config):
     if args.mode == "crossvalidation":
         data = training_data.load_data(args.data)
         data = prepare_data(data, cutoff = 5)
-        results = run_cv_evaluation(data, int(args.folds), nlu_config)
+        _,results = run_cv_evaluation(data, int(args.folds), nlu_config)
```

Comment:
Mhm, aren't we ignoring the results of the evaluation on the train set here? I mean in that case we wouldn't need to do it in the first place.